src/main/java/org/opensearch/geospatial/geojson/Feature.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.geospatial.geojson;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Feature object represents GEOJSON of type Feature.
+ */
+public class Feature {
+    public static final String TYPE = "Feature";
+    public static final String GEOMETRY_KEY = "geometry";
+    public static final String PROPERTIES_KEY = "properties";
+    public static final String TYPE_KEY = "type";
+    private final Map<String, Object> geometry;
+    private Map<String, Object> properties;
+
+    private Feature(Map<String, Object> geometry) {
+        this.geometry = new HashMap<>();
+        this.geometry.putAll(geometry);
+        this.properties = new HashMap<>();
+    }
+
+    /**
+     * Gets the type value
+     *
+     * @return Feature's type value
+     */
+    public String getType() {
```

Comment:
Why do we need a getter on a public static type?

Reply:
Good catch. I forgot to remove this method after i made it as public static constant.